src/libs/common.go

```diff
@@ -438,7 +441,16 @@ func WriteKnoxNetPolicyToYamlFile(namespace string, policies []types.KnoxNetwork
 }
 
 func WriteCiliumPolicyToYamlFile(namespace string, policies []types.CiliumNetworkPolicy) {
-	fileName := GetEnv("POLICY_DIR", "./")
+
+	policyDir := cfg.CurrentCfg.ConfigNetPolicy.NetworkPolicyDir
+	var fileName string
+
+	if policyDir != "./" {
+		fileName = GetEnv("POLICY_DIR", "./")
+	} else {
+		fileName = policyDir
+	}
```

Comment:
This code does not make sense.
If the NetworkPolicyDir in the configmap is set to any value other thant "./" then it will not be used!!

Reply:
No Rahul. As per implementation, if it is any value other than **./**, then the same will be used.